src/nxt_conf_validation.c

```diff
@@ -2803,6 +2834,18 @@ nxt_conf_vldt_target(nxt_conf_validation_t *vldt, nxt_str_t *name,
 }
 
 
+#if (NXT_HAVE_CGROUP)
+
+static nxt_int_t
+nxt_conf_vldt_cgroup(nxt_conf_validation_t *vldt, nxt_conf_value_t *value,
+    void *data)
+{
+    return nxt_conf_vldt_object(vldt, value, data);
```

Comment:
Do you expect to change this?  Otherwise, it might be simpler to just `.validator  = nxt_conf_vldt_object,`

Reply:
Right, I just copied what the namspaces was doing. Currently "cgroup" just a single option, in future it could grow to include controllers for setting resource limits. But yes, for now it can probably just use nxt_conf_vldt_object.